pycon/finaid/models.py

```diff
@@ -236,3 +236,32 @@ class FinancialAidEmailTemplate(models.Model):
 
     def __unicode__(self):
         return self.name
+
+
+def user_directory_path(instance, filename):
+    """
+    Method for finding the directory path to upload a receipt to.
+
+    This method should live inside of the Receipt model, but support for this
+    behavior is only added in python3. See the following for more info:
+    https://docs.djangoproject.com/en/1.7/topics/migrations/#serializing-values
+    """
+    # file will be uploaded to MEDIA_ROOT/finaid_receipts/<user>/<filename>
+    return 'finaid_receipts/{}/{}'.format(instance.user.username, filename)
+
+
+class Receipt(models.Model):
+    timestamp = models.DateTimeField(auto_now_add=True, editable=False)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
+                  related_name='financial_aid_receipts')
+    application = models.ForeignKey(FinancialAidApplication,
```

Comment:
Actually the application already has a 'user' field so we probably don't need a user field here too.  (And so we don't have to worry about the two fields getting out of sync.)